src/codegen/program.lisp

```diff
@@ -165,6 +165,14 @@ A function bound here will be called with a keyword category, and one or more ad
                       ,name
                       ,(codegen-expression node nil env)))
 
+  ;; Print types of definitions
+  (when (or *compile-print* settings:*compile-print-types*) 
+    (dolist (binding bindings)
```

Comment:
Do we want to print the package prefix of the symbol being defined? Probably not...